### ISSUE TITLE
BUG: validation is skipped when we using curly braces in regexp

### ISSUE THREAD

**Issue by bieli** [go-validator/validator#56]
Example pattern in YAML validation:
```
Listen string `yaml:"Listen,omitempty" validate:"regexp=^(([0-9]+[.][0-9]+[.][0-9]+[.][0-9]+)?[:][0-9]\{1,6\})$"`
```
temporary solution in my configuration:
```
Listen string `yaml:"Listen,omitempty" validate:"regexp=^(([0-9]+[.][0-9]+[.][0-9]+[.][0-9]+)?[:][0-9]+)$"`
```

**Comment by emaraschio** [comment]
Hello @bieli 
Did you tried without the escape character?
something like this: `(([0-9]+[.][0-9]+[.][0-9]+[.][0-9]+)?[:][0-9]{1,6})$`

I don't know exactly if you need the `\`
Here is a good tool to check regex -> https://regex-golang.appspot.com/assets/html/index.html

Hope it helps!

**Comment by jameshartig** [comment]
@bieli does that fix it?

**Comment by aashayshah** [comment]
@emaraschio Without the escape character, it gives me an error "Unknown tag"
My code is here -> validate:"regexp=^[a-zA-Z .]\{1,100\}"
Which gave me an runtime error"Unknown tag". Is this a bug?

**Comment by aashayshah** [comment]
@emaraschio Also, after adding the escape character, it just skips the validation as mentioned by @bieli .
I used the validator as  validate:"regexp=^[a-zA-Z \\.]{1,100}" which does not give me the error "Unknown tag" as earlier,  but now it just bypasses even if I supply gibberish like "@!23#$R".

**Comment by jameshartig** [comment]
@aashayshah the comma problem was fixed with #52. You need to send `validate:"regexp=^[a-zA-Z .]{1\\,100}"` (notice the escape of the comma).
